.travis.yml

```diff
@@ -12,6 +11,9 @@ php:
 matrix:
     include:
         - php: 5.3
+          dist: precise
```

Comment:
I think one job for PHP 5.3 should be enough

Reply:
Which one should I keep? with lowest dependencies?

Reply:
Don't know what @stof prefers. I think I would run the lowest dependencies build with PHP 7.